canonical-web-and-design/buy.ubuntu.com
# buy.u.c clean up

## Done

- Fixed HTML validation errors where possible. In some cases, Shopify itself introduces validation errors with `<meta>` tags, via CMS content.
- Added Vanilla patterns to remaining pages
- Shopify provides pages at /search, /blogs/news and /products/{product.slug}. These have been Vanilla-ised should it be decided they're needed in future, but for the time being they all now redirect to the homepage.
- Refactored templates so that site nav and the `<main>` element are not repeated in each template.

## QA
- Visit https://canonical-staging.myshopify.com/hello
- See that the resulting 404 page is not broken.
- Visit:
  - https://canonical-staging.myshopify.com/search
  - https://canonical-staging.myshopify.com/blogs/news
  - https://canonical-staging.myshopify.com/products/standard-server
- See that you are redirected to the homepage
- Visit:
  - https://validator.w3.org/nu/?doc=https%3A%2F%2Fcanonical-staging.myshopify.com%2F
  - https://validator.w3.org/nu/?doc=https%3A%2F%2Fcanonical-staging.myshopify.com%2Fcart
  - https://canonical-staging.myshopify.com/collections/ubuntu-advantage-for-virtual-guests
- See that any errors or warnings relate to meta and script tags injected by Shopify

Fixes #21
